sebaztyn/Property-Pro-Lite
# #167268891 refactor validators - CHALLENGE 3

#### What does this PR do?
refactor the endpoint validators for Property Pro Lite
#### Description of Task to be completed?
- install express-validator module
- write the validators for all endpoints.
- run tests on endpoint with different edge cases and monitor responses to validators.

#### How should this be manually tested?
 - clone repository
- startup terminal
- cd into the cloned repository
- start server by typing ```npm start``` into the terminal
- Enter wrong values into postman and see result.

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167268891
#### Screenshots (if appropriate)
None
#### Questions:
None
